core/src/test/java/org/elasticsearch/search/aggregations/pipeline/InternalSimpleValueTests.java

```diff
@@ -55,4 +55,16 @@ protected void assertReduced(InternalSimpleValue reduced, List<InternalSimpleVal
         return InternalSimpleValue::new;
     }
 
+    @Override
+    protected void assertFromXContent(InternalSimpleValue simpleValue, ParsedAggregation parsedAggregation) {
+        ParsedSimpleValue parsed = ((ParsedSimpleValue) parsedAggregation);
+        if (Double.isInfinite(simpleValue.getValue()) == false && Double.isNaN(simpleValue.getValue()) == false) {
+            assertEquals(simpleValue.getValue(), parsed.value(), Double.MIN_VALUE);
```

Comment:
The version without a delta is deprecated, so I think better to add one. @tlrx yes, I can use 0 here, but I usually prefer to leave a very small value to allow minimal rounding errors. But I will change the delta to 0 here, hope to remember that when I see it in other place, please remind me if I don't ;-)